textile/features.textile

```diff
@@ -2590,6 +2593,7 @@ class ObjectMessage // OM*, internal
   operation: ObjectOperation? // OM2f
   object: ObjectState? // OM2g
   serial: String // OM2h
+  serialTimestamp: Number // OM2j
```

Comment:
Is the correct type here not a `Time`, given that we're meant to potentially compare it to a locally-generated time per RTLM19a1?